daemon/src/engine/client/cg_api.h

```diff
@@ -213,6 +215,8 @@ void            trap_Key_SetBinding( int keyNum, int team, const char *cmd );
 void            trap_Key_ClearCmdButtons( void );
 void            trap_Key_ClearStates( void );
 std::vector<int> trap_Key_KeysDown( const std::vector<int>& keys );
+bool            trap_Mouse_GetCursorMode();
```

Comment:
Can this be renamed to IsCursorVisible/Active? Mode seems to imply that there are multiple states but this just seems to hide/show the cursor.